src/services/userService.js

```diff
@@ -30,6 +30,24 @@ const findFollower = async (followeeId, followerId) => {
   return follower;
 };
 
+/**
+ * @name getAllUsers
+ * @returns {Array} array of users
+ */
+const getAllUsers = async () => {
+  const allUsers = await User.findAll();
+  const users = allUsers.map(user => ({
```

Comment:
instead of mapping why not select only the fields you want using sequelize using {attributes: ['firstName', 'lastName', ...]}

Reply:
I didn't know that was possible, I've made the changes. Thanks for the feedback!